src/GraphQL/GraphQL.csproj

```diff
@@ -29,6 +29,7 @@
     <PackageReference Include="GraphQL-Parser" Version="3.0.0" />
     <PackageReference Include="Newtonsoft.Json" Version="10.0.3" />
     <PackageReference Include="System.Buffers" Version="4.5.0" />
+    <PackageReference Include="System.ComponentModel.Annotations" Version="4.5.0" />
```

Comment:
Is it permissible to add this dependency?

Reply:
Admittedly it makes me cringe a bit but that is probably the easiest solution.

Reply:
Well, this is essentially standardized well-known library with common meta-information. If not this, then what? I do not really want to reinvent the wheel.